packages/components/src/guide/page.tsx

```diff
@@ -4,7 +4,14 @@
 import { useEffect } from '@wordpress/element';
 import deprecated from '@wordpress/deprecated';
 
-export default function GuidePage( props ) {
+/**
+ * Internal dependencies
+ */
+import type { WordPressComponentProps } from '../ui/context';
+
+export default function GuidePage(
+	props: WordPressComponentProps< {}, 'div' >
```

Comment:
```suggestion
	props: WordPressComponentProps< {}, 'div', false >
```

This also needs a `false` as the third argument because this component isn't polymorphic (i.e. doesn't accept an `as` prop).